Makefile

```diff
@@ -313,6 +313,7 @@ site-check: ## Test the site's links
 
 integration: ## Run integration tests against a real k8s cluster
 	./_integration/testsuite/make-kind-cluster.sh
+	./_integration/testsuite/install-fallback-certificate.sh
```

Comment:
Can the test apply these bits? This might get out of hand as we add more test logic.

Reply:
Not really. I discussed this on slack .. the problem is that we have no way to roll back the change that we make to the contour deployment. It's theoretically possible to do that, but also pretty involved.

Reply:
But yes, trying to test different Contour install configs will rapidly get out of hand :-(